libs/types/src/fixed_point.rs

```diff
@@ -743,6 +745,15 @@ impl<const DIV: u128> One for FixedU128<DIV> {
 	}
 }
 
+impl<const DIV: u128> SaturatedProration for FixedU128<DIV> {
+	type Time = Seconds;
+
+	fn saturated_proration(annual_rate: Self, period: Self::Time) -> Self {
+		let time = period.saturating_div(SECONDS_PER_YEAR);
+		annual_rate.saturating_mul(Self::from_integer(time.into()))
+	}
+}
+
```

Comment:
This seems a domain-related method. There is a type `InterestRate` in `libs/trait/src/interest.rs` that may be useful

Reply:
Good catch. This is the saturated version of `InterestRate::per_sec()` multiplied with the epoch duration in seconds. Not sure how to unite both. For pool fees, there is no compounding. So it does not seem feasible to me to use the `InterestRate` type for the [`PoolFeeAmount`](https://github.com/centrifuge/centrifuge-chain/blob/935082685544fb456432e6675f6c09eb15f9bb81/libs/types/src/pools.rs#L192-L200). 

Do you have a suggestion?

Reply:
Mmm... I see. I have no strong opinion, then. Maybe this should be treated as a utility (conversion?) function instead of a trait method.